<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-NE7DH0T6" ignored="3" name="Default test" tests="4" failures="2" timestamp="12 Jun 2020 23:34:18 GMT" time="111.465" errors="0">
  <testcase name="CheckInDateIsLarger" time="0.431" classname="InvokingMethods.InvokingMethodsForValidLogin"/>
  <testcase name="InvalidDate" time="0.024" classname="InvokingMethods.InvokingMethodsForValidLogin"/>
  <testcase name="ValidCase" time="36.757" classname="InvokingMethods.InvokingMethodsForValidLogin">
    <failure type="java.lang.AssertionError" message="C:\Users\Apoorv Sharma\workspace\cognizant\Apoorv_Sharma_849341\OutputFile.xlsx (The system cannot find the path specified)">
      <![CDATA[java.lang.AssertionError: C:\Users\Apoorv Sharma\workspace\cognizant\Apoorv_Sharma_849341\OutputFile.xlsx (The system cannot find the path specified)
at org.testng.Assert.fail(Assert.java:96)
at ExtentReports.ReportMethods.reportFail(ReportMethods.java:17)
at SearchHolidayHomes.ResultList.Results(ResultList.java:58)
at InvokingMethods.InvokingMethodsForValidLogin.ValidCase(InvokingMethodsForValidLogin.java:45)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValidCase -->
  <testcase name="InvalidCase" time="69.514" classname="InvokingMethods.InvokingMethodsForInvalidLogin">
    <failure type="java.lang.AssertionError" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;/html/body/div[2]/div/div[1]/div[3]/div/div[1]/span&amp;quot;}
  (Session info: chrome=83.0.4103.97)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:25:48&amp;apos;
System info: host: &amp;apos;LAPTOP-NE7DH0T6&amp;apos;, ip: &amp;apos;192.168.17.1&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_241&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 83.0.4103.97, chrome: {chromedriverVersion: 83.0.4103.39 (ccbf011cb2d2b..., userDataDir: C:\Users\APOORV~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:53299}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 26bb2f0fcebd5ffbe33aece4237a72de
*** Element info: {Using=xpath, value=/html/body/div[2]/div/div[1]/div[3]/div/div[1]/span}">
      <![CDATA[java.lang.AssertionError: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[2]/div/div[1]/div[3]/div/div[1]/span"}
  (Session info: chrome=83.0.4103.97)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'LAPTOP-NE7DH0T6', ip: '192.168.17.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_241'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 83.0.4103.97, chrome: {chromedriverVersion: 83.0.4103.39 (ccbf011cb2d2b..., userDataDir: C:\Users\APOORV~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:53299}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 26bb2f0fcebd5ffbe33aece4237a72de
*** Element info: {Using=xpath, value=/html/body/div[2]/div/div[1]/div[3]/div/div[1]/span}
at org.testng.Assert.fail(Assert.java:96)
at ExtentReports.ReportMethods.reportFail(ReportMethods.java:17)
at SignIn.InvalidCredentials.InvalidLogin(InvalidCredentials.java:51)
at InvokingMethods.InvokingMethodsForInvalidLogin.InvalidCase(InvokingMethodsForInvalidLogin.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- InvalidCase -->
  <testcase name="@AfterTest closebrowser" time="0.0" classname="DriverSetup.DriverSetup">
    <ignored/>
  </testcase> <!-- @AfterTest closebrowser -->
  <testcase name="@AfterTest closebrowser" time="0.0" classname="DriverSetup.DriverSetup">
    <ignored/>
  </testcase> <!-- @AfterTest closebrowser -->
  <testcase name="@AfterTest closebrowser" time="0.0" classname="DriverSetup.DriverSetup">
    <ignored/>
  </testcase> <!-- @AfterTest closebrowser -->
</testsuite> <!-- Default test -->
